app/connect_db.py

```diff
@@ -21,6 +21,9 @@ def update_language_target_for_user(user_id, language):
 
 
 def add_new_user(user_id, language_target="en", language_from="ru"):
+    print(user_id)
```

Comment:
this is fine for now. though generally you should be using module for printing purpose, we will be doing it in the next project